blog/content/post/2020-12-21-cicd-tools-comparison.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Comparing popular CI/CD tools for on-premise configuration'
+abstract: 'Comparing CI/CD tools: Drone, Buildbot, Tekton, Concourse.
+           Self-hosted and open-source alternatives to Jenkins'
+cover: /covers/image-file.png
+author: piotr.konkol
+layout: post
+published: true
+date: 2020-12-21
+archives: "2020"
+
+tags:
+  - devops
+  - ci
+  - cd
+  - constant-integration
+  - contant-delivery
+  - infrastructure
+  - on-premise
+  - self-hosted
+categories:
+  - App Dev
+
+---
+
+As constant integration/constant delivery workflow has grown in popularity in
+recent years a multitude of tools intended for simplifying this task. As we
+are supportive and passionate for open-source and self-hosted solutions we
+will compare some of the most interesting projects available that may act
+as an alternative to the most popular one, Jenkins or the multitude of
+proprietary tools.
+
+## Drone.io
+
+* Github stars: 22.3k
```

Comment:
Links to github, homepage, documentation and other references would be useful. Now I would have to google it myself to verify some stuff or learn more about the project.